### PR TITLE
Center the support footer

### PR DESCRIPTION
**Pull request by MichaelZetune**
### Description

The footer at the bottom of CiviForm has been off-center for a while. This PR fixes that.

#### New Features

- [x] Adjust Tailwind classes and divs to make the footer horizontally centered.
- [x] Update screenshots

### CODE REVIEW

File: server/app/views/applicant/ApplicantLayout.java [hunk 1]
```diff
@@ -95,11 +95,12 @@ private Content renderWithSupportFooter(HtmlBundle bundle, Messages messages) {
...
         div()
+            .withClasses("flex", "flex-row")
             .with(
-                text(messages.at(MessageKey.FOOTER_SUPPORT_LINK_DESCRIPTION.getKeyName())),
-                text(" "),
-                emailAction)
-            .withClasses("mx-auto", "max-w-screen-sm", "w-5/6");
+                div()
+                    .withText(messages.at(MessageKey.FOOTER_SUPPORT_LINK_DESCRIPTION.getKeyName()))
+                    .with(text(" "), emailAction)
```
Comment (reemaka) [inline server/app/views/applicant/ApplicantLayout.java:102]: nit: if it doesn't affect the UI, i think the old way of having everything in one with() is a bit cleaner.

**Review by reemaka — APPROVED** [review]

File: server/app/views/applicant/ApplicantLayout.java (hunk 1 above)
Comment (MichaelZetune) [reply server/app/views/applicant/ApplicantLayout.java:102]: It does affect the UI, the div was needed to get this centered afaict.

Comment (reemaka) [reply server/app/views/applicant/ApplicantLayout.java:102]: I meant keeping the inner div, but going from
```
div().withText(<message>).with(text(" "), emailAction)
```
to 
```
div().with(text(<message>), text(" "), emailAction)
```

Comment (MichaelZetune) [reply server/app/views/applicant/ApplicantLayout.java]: Gotcha, that worked 👍

**Pull request merged**